beacon-chain/attestation/service.go

```diff
@@ -121,6 +121,9 @@ func (a *Service) LatestAttestationTarget(ctx context.Context, index uint64) (*p
 	if err != nil {
 		return nil, fmt.Errorf("could not get attestation: %v", err)
 	}
+	if attestation == nil {
+		return nil, nil
```

Comment:
this is reasonable, if an attester did not attest - the attestation target should just be nil instead of err'ing on a skipped attestation